pkgs/tools/wayland/hyprland-iio/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, meson, cmake, pkg-config, dbus, ninja, ... }:
+
+stdenv.mkDerivation {
+  pname = "iio hyprland";
+  version = "1.0.0";
```

```suggestion
  version = "0-unstable-2023-09-27";
```
Upstream hasn't tagged a release